src/app/Models/FinancialOperation.php

```diff
@@ -64,6 +78,26 @@ public function scopeIncomes(Builder $query): Builder
             ->where('expense', '=', false);
     }
 
+    /**
+     * Extends a query asking for financial operations so that it demands only
+     * operations which represent unrepayed lendings.
+     *
+     * @param Builder $query
+     * the builder whose query to extend
+     * @return Builder
+     * the extended query builder
+     */
+    public function scopeUnrepayedLendings(Builder $query): Builder
```

Comment:
"Unrepaid", prípadne čitateľnejšie by bolo "Unpaid"

Reply:
Ehm...samozrejme, dám tam to "Unrepaid", keďže chcem zachovať tú sémantiku, že sú vybrané pôžičky, ktoré neboli splatené.